data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/StudentViewRepository.java

```diff
@@ -0,0 +1,24 @@
+package io.micronaut.data.jdbc.oraclexe.jsonview;
+
+import io.micronaut.data.annotation.Query;
+import io.micronaut.data.annotation.QueryResult;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.jdbc.annotation.JdbcRepository;
+import io.micronaut.data.model.DataType;
+import io.micronaut.data.model.query.builder.sql.Dialect;
+import io.micronaut.data.repository.PageableRepository;
+
+import java.util.Optional;
+
+@JdbcRepository(dialect = Dialect.ORACLE)
+@QueryResult(type = QueryResult.Type.JSON)
```

Comment:
do we still need this annotation? Can we not infer this from the fact the entity is annotated with `@JsonView`?

Reply:
With some additional changes, it is not required now. Thanks for the suggestion.